src/components/Stepper/index.js

```diff
@@ -255,10 +234,7 @@ const Stepper = ({
         disabled={isPlusDisabled}
         aria-label="Plus"
       >
-        <StyledSeparator size={size} style={{ left: 0 }}>
-          |
-        </StyledSeparator>
-        <Icon name="plus" size={size === 'large' ? 28 : 18} color="gray300" />
+        <StyledIcon name="plus" size={iconSizes[size]} />
```

Comment:
Why `minus` icon has not the same size prop ??

Reply:
my bad I forgot to change minus icon too